source/_integrations/romy.markdown

```diff
@@ -35,6 +35,20 @@ Currently supported services are:
 - `stop`
 - `return_to_base`
 
+## Integration entities
+
+The **Romy** {% term integrations %} will add the following sensors.
```

Comment:
- This is not the first time this term has been introduced.
- No need to emphasize the brand name here?
- 
```suggestion
The Romy will add the following sensors.
```